frontend/src/components/data-table/charts/components/form-fields.tsx

```diff
@@ -360,11 +360,6 @@ export const SliderField = ({
   const [internalValue, setInternalValue] = React.useState(value);
   const form = useFormContext();
 
-  // Update internal value on prop change
-  React.useEffect(() => {
-    setInternalValue(value);
-  }, [value]);
```

Comment:
are you sure removing this is ok? what happens if the slider is updated itnernally?